test/pkg/environment/common/environment.go

```diff
@@ -58,6 +58,9 @@ type Environment struct {
 	Monitor    *Monitor
 
 	StartingNodeCount int
+
+	ClusterName     string
```

Comment:
These are AWS-specific and may not be part of the common environment practically, we should move this over to the AWS environment

Reply:
Sounds good, I honestly wasn't sure where they should live 👍